### DISCUSSION TITLE
Sidebar example?

### DISCUSSION THREAD

**Discussion by DMcP89** [emad-elsaid/xlog#71] [Q&A]
Is there an example of how to setup a site with the sidebar? I have the sidebar flag set to true but do not see it while running xlog directly or generating a static site.

**Comment by emad-elsaid** [comment]
Hello @DMcP89 , the sidebar was removed long time ago from xlog, I kept the flag to keep old setups from breaking when specifying `--sidebar=false` in their commands. I removed this flag yesterday in preparations for another major release.

however xlog allow developing extensions and has all the APIs needed to do a sidebar extension if you have the time for that it can be a fun side project :smiley:
